JP-3824: Fix a bug in wcs_from_sregions when crpix is provided (#326)
spacetelescope/stcal: changes/326.bugfix.rst

> Fix a bug in ``alignment.util.wcs_from_sregions()`` and
> ``alignment.util.wcs_from_footprints()`` functions due to which, when the
> ``crpix`` argument was provided, the bounding box of the created WCS was
> not adjusted to account for this ``crpix`` value.

Reviewer: ```suggestion
Fix a bug in ``alignment.util.wcs_from_sregions()`` and
``alignment.util.wcs_from_footprints()`` functions that was causing incorrect WCS bounding boxes
when the ``crpix`` argument was provided.
```
